### PR TITLE
[Term Entry] discrete-math/proofs/conditional-proof.md

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

-Added term **Conditional Proof** under concept _proofs_ within _discrete-math_.

Motivation:

The concept _proofs_ contains a term for one kind of proof technique within discrete math (namely, proof by strong induction), so I figured it would be helpful to have terms for other common proof techniques within discrete math.

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon I did one small edit, and it looks good for a second review.

**Comment by yangc95** [comment]
@noahpgordon currently the terms table under proofs covers the various methods of proofs. I'm wondering if "conditional proof" falls under these different methods (like proof by induction, cases, contradiction, etc.) or rather if it applied to all proofs.

Could you help walk me though what you thinking is? Thanks in advance 🙂 cc: @HishamT

**Comment by noahpgordon** [comment]
> @noahpgordon currently the terms table under proofs covers the various methods of proofs. I'm wondering if "conditional proof" falls under these different methods (like proof by induction, cases, contradiction, etc.) or rather if it applied to all proofs.
> 
> Could you help walk me though what you thinking is? Thanks in advance 🙂 cc: @HishamT

Conditional proof is a method of proving a specific type of statement, specifically a conditional statement. The same can be said for other proof methods. For example, proof by induction is a method of proving specifically a universal generalization. Not every proof uses conditional proof, since not every proof involves a conditional statement.

Note also that a single proof can encompass multiple proof methods. For example, you might have an induction that uses proof by contradiction or proof by cases to establish the base step. Also, inductive proofs tend to use conditional proof for the inductive step, although this is generally not explicitly stated.

Does this help answer your question?

**Review by HishamT — CHANGES_REQUESTED** [review]
I feel like the language in this entry is a little advanced.

File: content/discrete-math/concepts/proofs/terms/conditional-proof/conditional-proof.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Arithmetic'
+  - 'Logic'
+  - 'Logical'
+  - 'Math'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Conditional Proof** is a technique used in mathematical proofs to demonstrate a conditional statement, by assuming the antecedent of the conditional and showing that the consequent follows.
```
[the remark this answers is not in the document]
Comment (noahpgordon) [reply content/discrete-math/concepts/proofs/terms/conditional-proof/conditional-proof.md]: I moved the description of the technique below the definition of relevant terms ('antecedent' and 'consequent'). Hope this improves comprehension.

**Review by HishamT — APPROVED** [review]
LGTM

**Pull request merged**